installer/x86_64/install.sh

```diff
@@ -92,6 +92,39 @@ VAR_LOG_SIZE=4096
 
 [ -r platforms/$onie_platform ] && . platforms/$onie_platform
 
+# Verify image platform is inside devices list
+# Temporarily set TIMEOUT as 10 seconds
+if [ "$install_env" = "onie" ] && [ -d devices ]; then
```

Comment:
>-d devices [](http://example.com/codeflow?start=36&length=10)

This should not happen according to your code. If you want to handle this case, it should be an immediate failure.

Reply:
That's true. I was thinking the wrong way.